## Request Constants
uint8 N = 1
uint8 NE = 2
uint8 E = 3
uint8 SE = 4
uint8 S = 5
uint8 SW = 6
uint8 W = 7
uint8 NW = 8

# Request Fields
int8 direction
